src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const i = days;
```

Comment:
whu do you nee this variable? (and also if you create variables - use more meanfull names)